.env.example

```diff
@@ -441,7 +441,7 @@ LDAP_URL=
 LDAP_BIND_DN=
 LDAP_BIND_CREDENTIALS=
 LDAP_USER_SEARCH_BASE=
-LDAP_SEARCH_FILTER=mail={{username}}
+#LDAP_SEARCH_FILTER="mail="
```

Comment:
why change this?

Reply:
Because `{{username}}` in an ansible template leads to an error if the `username` variable is not defined. So I suppose it's a leftover of somebody pushing a `.env.example` with some ansible leftovers.

Github doesn't seem to like the git commit, but you can find it here:

```bash
git diff -R a61826690560d4e88c3a0edb52b281798c5cf5a4..d5a7806e3286a946f4e91d29c0e177b2e14d443b
```

It is the original commit adding LDAP to librechat.